app/src/pages/movies/movie/components/DeleteMovieForm.jsx

```diff
@@ -21,19 +23,25 @@ const ButtonContainer = styled.div`
 `
 
 function DeleteMovieForm(props) {    
+    function confirmDelete(event){
+        event.preventDefault();

```

Comment:
event.preventDefault(); 

 should it be everywhere?

Reply:
Fixed: place `event.preventDefault();` to common `FormComponent`